package.json

```diff
@@ -13,6 +13,7 @@
     "dev": "npx rollup -c --watch",
     "build": "npx yarn start && npx yarn timeseries && npx yarn buildSite",
     "buildSite": "npx rollup -c",
+    "dataValidation": "jest tests/*validation.*",
```

Comment:
I would suggest including the `dataValidation` as part of the `build` or `deploy` script to make sure it gets run.

Reply:
we definitely need some sort of report that can be run during `build` or `deploy`! Going to look into `jest-reporting` options and will update this ticket with what I find